ledger-core/ledger/jet/prefix_tree_slow_test.go

```diff
@@ -3,7 +3,7 @@
 // This material is licensed under the Insolar License version 1.0,
 // available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
 
-// +build slowtest
+// +build longtest
```

Comment:
long tests are to be executed multiple times
this test is just veeeery slow, but it doesn't need to be repeated
so should be either
// +build slowtest,!longtest 
or something else

Reply:
Ok, I'll make another tag and will add another test run to this workflow